components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java

```diff
@@ -88,12 +93,17 @@
                                     field("connectors", serverConnectorsSchema),
                                     optional("bossThreadsCount", integer()),
                                     optional("workerThreadsCount", integer()),
+                                    // tcpNoDelay is deprecated by PR #464
                                     optional("tcpNoDelay", bool()),
+                                    // nioReuseAddress is deprecated by PR #464
                                     optional("nioReuseAddress", bool()),
+                                    // nioKeepAlive is deprecated by PR #464
                                     optional("nioKeepAlive", bool()),
+                                    // maxInitialLength is deprecated by PR #464
```

Comment:
Also here. Remove the comment. `maxInitialLength` is still relevant.

Reply:
Done